### DISCUSSION TITLE
I need access to commit to the main directory

### DISCUSSION THREAD

**Discussion by ghost** [ASEAN-Build-The-Earth/Merlion#55] [Q&A] [answered]
I need access to commit my changes to the code to the main directory

**Comment by StoneMcYT** [comment, marked as answer]
We can't give that to you. Since you are not an official contributor. Also please don't be so ignorant and actually read the basics for once.
https://github.com/ASEAN-Build-The-Earth/Merlion/issues/29

**Reply by ghost** [reply]
ok sorry about that

**Reply by ghost** [reply]
but like i need to make a pull request to the code so that i can commit the changes

**Reply by StoneMcYT** [reply]
> but like i need to make a pull request to the code so that i can commit the changes

Please read the linked issue before posting.

**Reply by StoneMcYT** [reply]
If you actually read it you couldn't have missed this.
```
after you are fine with the edit you do, create a pull request and compare it to the main with your forked repository here.
4.1 PLEASE check all you typo and code error before creating a pull request
wait for us to review your code and accept your pull request!
```

**Reply by StoneMcYT** [reply]
@S4ge-Rites  do you need me to explain it more?
